fluidd/README_en

```diff
@@ -1,18 +1,36 @@
-Instruction
+# Instruction for installing fluidd on K1/K1Max 3D printers
 
-Put the same level directory file ( fluidd ) in the root directory of the U disk
+## prerequisite 
+enable root access for your K1/K1Max printer in the settings menu of your touch screen display
 
-Copy the files ( fluidd.sh and fluidd.tar ) to directory ( /usr/data )
-cp /tmp/udisk/sda1/fluidd/* /usr/data/
+## get and install fluidd
+
+ssh as root:
+
+```
+$ ssh root@192.168.1.x
+root@192.168.1.x's password:  # default password is creality_2023 
+root@K1Max-1 /root [#] cd /usr/data/
+root@K1Max-1 /usr/data [#] wget https://raw.githubusercontent.com/CrealityOfficial/K1_Series_Annex/main/fluidd/fluidd/fluidd.sh
+root@K1Max-1 /usr/data [#] wget https://raw.githubusercontent.com/CrealityOfficial/K1_Series_Annex/main/fluidd/fluidd/fluidd.tar
+root@K1Max-1 /usr/data [#] chmod 744 fluidd.sh
+```
 
 Install fluidd, moonraker and nginx
-/usr/data/fluidd.sh install
+```
+root@K1Max-1 /usr/data [#] ./fluidd install
```

Comment:
Should not it be ?
`./fluidd.sh install`

Reply:
![image](https://github.com/CrealityOfficial/K1_Series_Annex/assets/56121322/62ee5ffa-631e-4343-a8ba-0d5bc8361536)
At least this is how it works in the Ubuntu terminal. I did not try the Putty option.